crates/biome_cli/src/execute/mod.rs

```diff
@@ -508,6 +513,21 @@ pub fn execute_mode(
                 };
                 reporter.write(&mut GithubReporterVisitor(console))?;
             }
+            ReportMode::GitLab => {
+                let reporter = GitLabReporter {
+                    summary: summary_result,
```

Comment:
It seems you don't use the `summary_result` inside `GitLabReporter`. If not used, there's no need to pass it